joomla/joomla-cms
# [4.0] Namespace Content - Confirm Consent plugin

### Summary of Changes

Namespaces Content - Confirm Consent plugin.

### Testing Instructions

Apply patch. Delete `libraries/autoload_psr4.php` file.
Enable Content - Confirm Consent plugin.
Create a contact with email.
View the contact form in frontend.
Check that consent checkbox appears in the form.

### Documentation Changes Required

IDK.
